### ISSUE TITLE
Parent Folder Hierarchy & Single Album Thumbnails

### ISSUE THREAD

**Issue by bhost909** [gmanley/loggia#4]
The order in which parent folders are listed seems to be a little strange, for example in here: https://love9.us/albums/2012-gghkfc-performances-2012-dot-07-dot-02-k-pop-nation-in-macao-sumyi

The hierarchy listed in the top goes:

```
Home /
2012 /
GGHKFC /
Performances /
[2012.07.02] K-Pop Nation in Macao /
sumyi /
```

While, if it were in the order the albums/subcategories are in, it should be:

Home -> Performances -> 2012 -> [2012.07.02] K-Pop Nation in Macao -> GGHKFC -> sumyi

Also, for albums with only a single image, (examples are Heavenly Forest and Shining smile in here: https://love9.us/categories/2012-performances-2012-dot-07-dot-02-k-pop-nation-in-macao), it seems like it's impossible to get a thumbnail image to correctly load/appear.

**Comment by gmanley** [comment]
Thanks for the heads up! I'll take a look into these two things.

**Comment by gmanley** [comment]
This should be fixed now with f07ad77
